packages/common/src/errors/error_messages.ts

```
@@ -4,3 +4,4 @@ export const ERROR_NOP = "nop";
 export const ERROR_WALLET_ADDRESS_FOR_CURRENCY = (currency: string) =>
     `This wallet address is already associated with another ${currency} wallet.`;
 export const ERROR_WALLET_ALREADY_ADDED_TO_ACCOUNT = "This wallet address is already associated with another wallet.";
+export const ERROR_TRANSACTION_ALREADY_ADDED = "A duplicate transaction has been found and the bacfil cannot be completed.";
```

Is this an error message that will be displayed to the user?

---

I created this error message from a backend perspective, I did not think about frontend implications. Did we plan on showing an error message to the user if the backfilling fails?

---

Also, yes the backfilling does occur during the adding of a wallet loading screen

---

Backfilling doesn't actually occur during the wallet loading screen right now because the method is async and not awaited.

---

I thought the loading screen display was handled by the frontend until the result of the call is returned.

---

It is but you are just basically creating a new thread and then proceeding / returning without waiting for the thread to complete, it isn't really a thread but the idea is similar

---

> I created this error message from a backend perspective, I did not think about frontend implications. Did we plan on showing an error message to the user if the backfilling fails?

Not that I'm aware of. I figured we'd silently fail/ignore them as @jason-gerard mentioned.